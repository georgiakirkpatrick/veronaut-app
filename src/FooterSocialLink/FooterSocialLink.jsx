import React from 'react'
import './FooterSocialLink.css'

const FooterSocialLink = props => {    
  const {
    id,
    pathToProfile,
    platform
  } = props

  return (
    <li className='Footer__social-link' key={id}>
      <a href={pathToProfile}>
        {platform}
      </a>
    </li>
  )
}

FooterSocialLink.defaultProps = {
  id: 1,
  pathToProfile: '',
  platform: ''
}

export default FooterSocialLink