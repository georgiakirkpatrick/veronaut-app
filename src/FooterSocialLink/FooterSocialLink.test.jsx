import React from 'react'
import { render, screen } from '@testing-library/react'
import { describe, it, expect } from 'vitest'
import FooterSocialLink from './FooterSocialLink'

describe('FooterSocialLink', () => {
  it('renders without crashing', () => {
    render(<FooterSocialLink />)
    screen.debug()
  })

  it ('renders the UI as expected', () => {
    const tree = render(<FooterSocialLink />)
    expect(tree).toMatchSnapshot()
  })
})    