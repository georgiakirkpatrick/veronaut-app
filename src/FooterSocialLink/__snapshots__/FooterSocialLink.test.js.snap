// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterSocialLink renders the UI as expected 1`] = `
<li
  className="Footer__social-link"
>
  <a
    href=""
  >
    
  </a>
</li>
`;
